app/services/legal_framework/proceeding_types/all.rb

```diff
@@ -37,7 +39,10 @@ def initialize(legal_aid_application)
       end
 
       def call
-        result = JSON.parse(request.body).map { |pt_hash| ProceedingTypeStruct.new(pt_hash) }
+        parsed_body = JSON.parse(request.body)
+        raise NoMatchingProceedingsFoundError, "nope!" if parsed_body.is_a?(Hash) && parsed_body["success"] == false
```

Comment:
😄, perhaps a "proceeding not found!"

Reply:
Do we need to test it is a hash? it was not being before but either way

Reply:
😬 that was a debug step that should have been removed! 🤦‍♂️

Reply:
The problem is that a successful return is an array. An unsuccessful result is a Hash, so `if parsed_body["success"]` will error on a successful call 😖

Reply:
spun out into new ticket